spring-cloud-gcp-samples/spring-cloud-gcp-logging-sample/pom.xml

```diff
@@ -69,13 +69,6 @@
             <plugin>
                 <groupId>com.google.cloud.tools</groupId>
                 <artifactId>appengine-maven-plugin</artifactId>
-                <version>1.3.2</version>
-            </plugin>
-            <plugin>
-                <artifactId>maven-deploy-plugin</artifactId>
-                <configuration>
-                    <skip>true</skip>
```

Comment:
Do we know for sure that the skip deploy configuration is inherited from the parent?
We really don't want to accidentally deploy samples to maven central.

Reply:
No, I don't know for sure if this works as intended. Maven n00b here :smile:

Reply:
I'm as much a n00b as you are on this.  :)
Maybe just run `mvn deploy` and see what happens?